File: examples/postcode/validate/go/returnARandomPostcodeAndItsInformation.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/postcode"
)

// Validate a postcode.
func ReturnArandomPostcodeAndItsInformation() {
	postcodeService := postcode.NewPostcodeService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := postcodeService.Validate(&postcode.ValidateRequest{
		Postcode: "SW1A 2AA",
	})
	fmt.Println(rsp, err)
}

File: examples/js/postcode/validate/returnARandomPostcodeAndItsInformation.js
const { PostcodeService } = require("m3o/postcode");

// Validate a postcode.
async function returnArandomPostcodeAndItsInformation() {
  let postcodeService = new PostcodeService(process.env.MICRO_API_TOKEN);
  let rsp = await postcodeService.validate({
    postcode: "SW1A 2AA",
  });
  console.log(rsp);
}

returnArandomPostcodeAndItsInformation();
